easybuild/easyconfigs/p/plotly.py/plotly.py-5.1.0-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'PythonBundle'
+
+name = 'plotly.py'
+version = '5.1.0'
+
+homepage = 'https://plot.ly/python'
+description = "An open-source, interactive graphing library for Python"
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+builddependencies = [
+    ('binutils', '2.36.1'),
+]
+
+dependencies = [('Python', '3.9.5')]
+
+use_pip = True
+
+exts_list = [
+    ('tenacity', '8.0.1', {
```

Comment:
@Micket just checking that it definitely doesn't need `retrying` anymore?

Reply:
The new tenacity sounds like something similar, and pip check isn't complaining about retrying missing (like it was for tenacity) so I think so

Reply:
Okay, as you say it would complain if there was something seriously wrong. Thanks.